stages.yml

```diff
@@ -47,20 +47,12 @@ parameters:
 - name: noarch
   type: boolean
   default: false
-- name: build_package
```

Comment:
so for the `ess-notebooks`, we were not building a C++ library and neither were we building a conda package nor running python tests.
Just building docs (which is why I had this parameter).
Not sure how we best handle the `ess-notebooks` builds. We basically just want to do a `sphinx-build`.

Reply:
So basically we *do* deploy, but only docs? And we want to use `sphinx-build` rather than `conda build`?

Reply:
I suppose the easiest option may be to figure out the details once we roll this out to `ess-notebooks`?

Reply:
Yes, there is nothing to 'conda build'. It just runs the `build_docs.py` script.
The tests is just to check that notebooks actually run without errors.